### PR TITLE
feat: add a view source toggle

### PR DESCRIPTION
**Pull request by russc**
On Friday, @adampash asked me to familiarize myself with this project and he mentioned "view source" and "edit source" as potential features.  This PR addresses the view source portion, but I wasn't sure where to put the button so it's just on top for now.

### CODE REVIEW

**Comment by adampash** [comment]
Oh cool. Rather than toggle b/w source and the rendering of the source, what if you just got a little scrollable `pre` `code` combo beneath the rendered text? Then it becomes a `view source`/`hide source` toggle. 

Also I think the toggle button makes sense below the story, and maybe a couple pixels smaller.

@ftrain thoughts?

**Comment by ftrain** [comment]
Adam's revision is good and I affirm it!

File: src/Section.js [hunk 1]
```diff
@@ -71,10 +83,40 @@ function Section(props) {
...
+  useEffect(() => {
+    if (viewSource) {
+      Prism.highlightAll();
+    }
+  }, [viewSource]);
+
   return (
     <div id="text">
-      <h1 key="h1">{props.page}</h1>
+      <h1 key="h1">{page}</h1>
```
Comment (adampash) [inline src/Section.js:98]: We don't need a key here, do we?

Comment (russc) [reply src/Section.js:98]: No we can remove the key

**Pull request merged**